tests/test_increased_belief_case.py

```diff
@@ -105,7 +105,11 @@ def test_that_update_for_a_linear_model(number_of_realizations):
             ]
         )
         A_posterior = ies.ensemble_smoother_update_step(
-            S, A, np.full(observations.shape, error), observations
+            S,
+            A,
+            np.full(observations.shape, error),
+            observations,
+            projected_response=S @ ies.response_projection(A),
```

Comment:
According to the paper, projection should not be necessary if the forward model is linear.
Why then is it needed here?